commands/deploy.go

```diff
@@ -35,6 +35,7 @@ type DeployCmd struct {
 	ServerURL   *url.URL      `default:"https://aperture.section.io/new/code_upload/v1/upload" help:"URL to upload application to"`
 	Timeout     time.Duration `default:"300s" help:"Timeout of individual HTTP requests."`
 	SkipDelete  bool          `help:"Skip delete of temporary tarball created to upload app."`
+	AppPath     string        `default:"nodejs" help:"Path of NodeJS application in environment repository."`
```

Comment:
I imagine the server API will protect against invalid characters in this argument but:

1. is the server error about why the apppath is wrong communication back to the user in a helpful form? and/or
1. should we validate the apppath locally and tell the user quickly instead of waiting until after server communication completes, especially the upload?